samples/github-samples.cabal

```diff
@@ -1,4 +1,4 @@
--- This file has been generated from package.yaml by hpack version 0.13.0.
+-- This file has been generated from package.yaml by hpack version 0.11.2.
```

Comment:
Ah, it seems you use old `stack`. Update it, IIRC `stack-1.1.0` has `hpack-0.13.0` bundled in